src/mesido/head_loss_class.py

```diff
@@ -410,29 +410,29 @@ def initialize_variables_nominals_and_bounds(
                 # possibilites. Line number 1, 2, N for the - & + side is created
                 discharge_type = ["neg_discharge", "pos_discharge"]
                 line_number = 0
-                for dtype in discharge_type:
-                    for ii_line in range(network_settings["n_linearization_lines"] * 2):
-                        if ii_line < network_settings["n_linearization_lines"]:
-                            dtype = discharge_type[0]
-                            line_number = ii_line + 1
-                        else:
-                            dtype = discharge_type[1]
-                            line_number = ii_line + 1 - network_settings["n_linearization_lines"]
-
-                        # start line segment numbering from 1 up to "n_linearization_lines"
-                        pipe_linear_line_segment_var_name = (
-                            f"{pipe_name}__pipe_linear_line_segment_num_{line_number}_{dtype}"
-                        )
+                # for dtype in discharge_type:
```

Comment:
Nice update. Commented out code to be delete: # for dtype in discharge_type:

Reply:
Deleted.